### DISCUSSION TITLE
Export functionality

### DISCUSSION THREAD

**Discussion by SilverServer34** [rappasoft/laravel-livewire-tables#745] [Q&A] [answered]
I have read in previous commits that an export feature was added a long time ago, ([commit](https://github.com/rappasoft/laravel-livewire-tables/commit/5001a90eb97e443eaf611b31a4a1955bb04ef1d8)) but I couldn't find anything in the documentation. 

Has it been deprecated in the meantime?
How can I integrate a csv, xslx and pdf export like those offered by Jquery Datatables?

**Comment by rappasoft** [comment, marked as answer]
Use the bulk actions functionality: https://rappasoft.com/docs/laravel-livewire-tables/v2/bulk-actions/creating-bulk-actions

You'll have to implement the actual logic using laravel-excel or some similar package though, but it should be straight forward.

**Reply by SilverServer34** [reply]
Thanks, I found the feature in another livewire datatables package, but I find your best all round and I was hoping to have the feature implemented already 😜  Patience I definitely prefer your implementation and all the care you put into it, congratulations.
